image_client/casbotany_sql_lite.py

```diff
@@ -394,6 +395,23 @@ def casbotany_lite_creator():
     curs.close()
     connect.close()
 
+def sql_lite_insert(sql, db_name):
```

Comment:
new function sql_lite_insert, analagous to insert_table_record, used to avoid having to add a testing mode to the insert_table_record function.